app/views/user_mentee_applications/new.html.erb

```diff
@@ -1,66 +1,83 @@
-<dl class="text-center mb-2">
-  <dt><%= current_user.full_name %></dt>
-</dl>
+<h1 class="text-center font-bold text-3xl mb-5">
+  Welcome <%= current_user.first_name %>!
+</h1>
 
-<%= form_with model: @user_mentee_application, local: true, html: { class: 'flex flex-col' } do |form| %>
+<%= form_with model: @user_mentee_application, local: true, html: { class: 'max-w-prose mx-auto' } do |form| %>
+  <div class='flex flex-col ml-5 mr-5'>
+    <%= render UserMenteeApplication::SectionComponent.new(header_text: 'Tell us about your journey') do %>
+      <div class="flex flex-col mb-5">
+        <%= form.label :learned_to_code, 'How did you learn to code?', class: 'font-bold' %>
+        <%= form.text_area :learned_to_code, class: 'form-input border-2 mb-2', required: true %>
```

Comment:
A lot of these inputs would look a bit better with some Daisy component styles in my opinion. Like [here's the stuff for textareas](https://daisyui.com/components/textarea/). And of course you can use their styles for all the other inputs as well. Can even maybe use the styles with colored borders if you think they look good, as injecting some color on this page would probably spice things up a bit.